### DISCUSSION TITLE
Fail to Display Bookmark in a Certain File

### DISCUSSION THREAD

**Discussion by zachho9** [sumatrapdfreader/sumatrapdf#4271] [General]
Hi devs, 

Just find that SumatraPDF could not show the bookmarks for a pdf, but Adobe Acrobat and Chrome could (Please see attached images).

PDF File download here: [ISLRv2_corrected_June_2023.pdf](https://github.com/sumatrapdfreader/sumatrapdf/files/15440969/ISLRv2_corrected_June_2023.pdf)

Adobe Acrobat screenshot:
![adobe](https://github.com/sumatrapdfreader/sumatrapdf/assets/49472689/f06f901a-04e9-403d-ad59-9dc8ffed6454)

Chrome screenshot:
![chrome](https://github.com/sumatrapdfreader/sumatrapdf/assets/49472689/7ff74be1-ff2d-4339-936d-800dd81be02c)

SumatraPDF screenshot:
![sumatra](https://github.com/sumatrapdfreader/sumatrapdf/assets/49472689/27d9a787-4e79-4715-b371-67ae3a9d177b)

I've been using SumatraPDF for many years and it works perfectly for me, love it!
Small issue, but I just want to let you know.
Thank you in advance.

**Comment by GitHubRulesOK** [comment]
Hmm think similar was mentioned before the file has problems however once fixed NEWER SumatraPDF does not see outline but 3.1.2 - 3.5.2  and other viewers do.

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/62421c87-a35d-49dc-9638-446c70ed369b)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/82045a53-d93b-4590-85ef-5e5c36af4a76)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/2ed083c1-0f1c-4c99-80d1-c8757d6aaae9)

**Reply by Back-T0** [reply]
Indeed, this type of documentation works fine under 3.5.2, forcing me to go back to 3.5.2. Even if it's a document problem, I don't bother fixing documents because I prefer click-and-go.

**Comment by GitHubRulesOK** [comment]
@kjk 

pre-rel from November worked but not February
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/c1f14eaf-2dd8-407a-9831-0c6c7a9b0b01)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/be9597a0-d028-499b-9929-15a6222c6e36)


This version from MuPDF correction works in latest !

[ISLRv2_corrected_June_2023 MuPDF.pdf](https://github.com/sumatrapdfreader/sumatrapdf/files/15441284/ISLRv2_corrected_June_2023.MuPDF.pdf)

**Reply by zachho9** [reply]
Sorry for the duplication and Thank you for the information and the fixed file! 
Since it's an issue with the file, I think it would better to leave it like this.

**Comment by kjk** [comment]
I've opened https://bugs.ghostscript.com/show_bug.cgi?id=707846 
If mupdf fixes it, we'll inherit the fix.

**Comment by CySop** [comment]
This was working for me too; but now it is not, same behavior as OP, except I'm on v3.5.2?

This is on two different files than OP:
	○ http://www.d6holocron.com/downloads/books/REUP.pdf
	○ http://d6holocron.com/downloads/books/GG16_The_Old_Republic.pdf

![image](https://github.com/user-attachments/assets/9b63b4ad-a680-4215-bcf6-0de0401ad196)

**Reply by GitHubRulesOK** [reply]
@CySop 
those are 2 different PDF types of file but REUP is not restricted type 1b and works "not a problem" in latest version so not a problem as per this bad file issue
![image](https://github.com/user-attachments/assets/6ad37cbe-1bcd-48ab-a61f-94bb826a61b5)

The other file has different PDF/A restrictions but fails to show the contents outline in Acrobat or any PDF reader so I dont think it has any valid bookmarks to show
![image](https://github.com/user-attachments/assets/3da45275-3f37-4b28-b4b0-d399f3e62d54)
